extensions/azure/azure-test/src/testFixtures/java/org/eclipse/dataspaceconnector/azure/testfixtures/CosmosTestClient.java

```diff
@@ -27,7 +28,7 @@
 
     static CosmosClient createClient() {
         var cosmosKey = propOrEnv("COSMOS_KEY", null);
-        if (cosmosKey != null) {
+        if (!StringUtils.isNullOrBlank(cosmosKey)) {
```

Comment:
is this changes needed here or part of a general improvement ? or do you expect key can be Blank now ?

Reply:
It wouldn't be anymore given pipeline changes, but could be if we add other test environments, so it seemed reasonable to leave the change as an improvement